docker_deploy/docker_manager_test.go

```diff
@@ -13,6 +13,21 @@ import (
 var imageName = "mcr.microsoft.com/dotnet/samples"
 var repoDigest = "sha256:d41fe80991d7c26ad43b052bb87c68a216a365c143623a62b5a5963fcdb77eb1"
 
+var options = map[string]interface{}{
+	"Hostname":   "my-container",
+	"User":       "root",
+	"Image":      "alpine",
+	"WorkingDir": "/app",
+	"StopSignal": "SIGTERM",
+}
+
+var hostOptions = map[string]interface{}{
+	"NetworkMode": "bridge",
+	"AutoRemove":  "true",
+	"Privileged":  "false",
```

Comment:
In the example you have a Bind element in this map, here you have Privileged and ShmSize. Are these just other possibilities? If so does the `docker_manager.go` need to include the parsing of these

Reply:
yeah these are other possibilities... there are many many more and we are basically committing to implementing them on an on request basis. 
Here are all of them.
https://pkg.go.dev/github.com/docker/docker@v26.1.1+incompatible/api/types/container#HostConfig